gui/wxpython/startup/utils.py

```diff
@@ -87,3 +87,33 @@ def create_database_directory():
         pass
 
     return None
+
+
+def get_startup_location():
```

Comment:
Since we may have a function which which tries to find existing user's copy/version/instance of the startup/demo location, I think this function name should be `get_startup_location_template()` to avoid confusion. The term _template_ is not used in GRASS, but it is common else where. Alternative is `get_startup_location_in_distribution()` which is what the function is doing, so perhaps better expressing the current that.

Reply:
Yep, get_startup_location_in_distribution() seems clearer. I used that.